using_openshift/executing_remote_commands.adoc

```diff
@@ -0,0 +1,74 @@
+= Executing Remote Commands
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+OpenShift takes advantage of a feature built into Kubernetes to support
+executing commands in containers. This is implemented using HTTP along with a
+multiplexed streaming protocol such as link:http://www.chromium.org/spdy[*SPDY*]
+or link:https://http2.github.io/[*HTTP/2*].
+
+== Client Operation
+Support for remote container command execution is
+link:../using_openshift/cli.html#common-cli-operations[built into the CLI]:
+
+****
+`$ osc exec -p _<pod>_ [-c _<container>_] -- _<command>_ _[<arg 1> ... <arg n>]_`
+****
+
+Clients initiate the execution of a remote command in a container by issuing a
+request to the Kubernetes API server:
+
+[options="nowrap"]
+----
+/proxy/minions/node123.openshift.com/exec/myns/mypod/mycontainer?command=date
+----
+
+In the above URL:
+
+- `_node123.openshift.com_` is the node name.
+- `_myns_` is the namespace of the target pod.
+- `_mypod_` is the name of the target pod.
+- `_mycontainer_` is the name of the target container.
+- `_date_` is the desired command to be executed.
+
```

Comment:
Might not be too big of a deal, but I feel the above could work just as well as replaceables in the actual filename string. Something like: "/proxy/minions/<Node_Name.openshift.com>/exec/<Namespace>/<Pod_Name>/<Container>?command=date"

This may become a problem with the <Container> being a target container and not the current container.